### ISSUE TITLE
Is it possible to use such browser for other test systems?

### ISSUE THREAD

**Issue by svetlyak40wt** [julian-baldwin/parachute-browser#2]
I prefer a Rove testing framework.

Is it possible to make this interface more generic and make plugins for other testing frameworks?

**Comment by julian-baldwin** [comment]
Yes, I think it would be possible to implement something more generic, although probably as a separate project. It was definitely something that crossed my mind while writing it initially, however I wanted a working solution for my own use as a first priority. As a result, it's really designed around Parachute and not easily generalisable.

**Comment by Shinmera** [comment]
The easier method would probably be to write a parachute simulation layer as exists for prove and 5am, and then inject that around the tests.

**Comment by julian-baldwin** [comment]
Agreed, similar to how I've wrapped DEFINE-TEST to provide source tracking. I think parachute, prove and 5am probably have the majority of the testing 'market' at the moment, unless you can think of any others that should be targeted?
